### PR TITLE
Target inline, use archived .html, .css and .tis files else use it from src dir

### PR DESCRIPTION
**Pull request by TopsDesk**
use the following command to create a compressed archive and use it with feature "inline" 
> packfolder ui resources.rc -i "*.html;*.css;*.tis" -v "resources" -binary

When feature inline is not defined then use resource files from "src" directory.  this helps in testing the tis, css and html files.

### CODE REVIEW

File: src/ui.rs [hunk 1]
```diff
@@ -30,9 +28,6 @@ struct UI(
 );
 
 fn get_msgbox() -> String {
-    #[cfg(feature = "inline")]
```
Comment (rustdesk) [inline src/ui.rs:33]: Why do you remove inline?

File: src/ui.rs [hunk 2]
```diff
@@ -1,6 +1,4 @@
 mod cm;
-#[cfg(feature = "inline")]
```
Comment (rustdesk) [inline src/ui.rs:2]: As we discussed, please do not modify inline feature.

File: Cargo.toml [hunk 3]
```diff
@@ -106,7 +106,8 @@ osx_minimum_system_version = "10.14"
 #!!! rembember call "strip target/release/rustdesk"
 # which reduce binary size a lot
 [profile.release]
```
Comment (rustdesk) [inline Cargo.toml:108]: Please do not modify this, it is not necessary.

File: Cargo.toml [hunk 4]
```diff
@@ -54,7 +54,7 @@ arboard = "2.0"
 
 [target.'cfg(target_os = "windows")'.dependencies]
 #systray = { git = "https://github.com/open-trade/systray-rs" }
-winapi = { version = "0.3", features = ["winuser"] }
+winapi = { version = "0.3", features = ["wincon","winuser"] }
```
Comment (rustdesk) [inline Cargo.toml:57]: comply with format please, there should be empty space.

File: src/ui.rs [hunk 5]
```diff
@@ -64,6 +59,14 @@ pub fn start(args: &mut [String]) {
         ALLOW_FILE_IO as u8 | ALLOW_SOCKET_IO as u8 | ALLOW_EVAL as u8 | ALLOW_SYSINFO as u8
     )));
     let mut frame = sciter::WindowBuilder::main_window().create();
+    #[cfg(feature = "inline")]
```
Comment (rustdesk) [inline src/ui.rs:129]: As we discussed, do not modify inline feature, you should define another feature. Your inline seems better than mine, but my inline has the advantage which we can modify html dynamically.

File: src/ui.rs [hunk 6]
```diff
@@ -115,19 +118,10 @@ pub fn start(args: &mut [String]) {
         log::error!("Wrong command: {:?}", args);
         return;
     }
+    /* fskhan 09-20-2021 with inline feature load from archive or load from directory */
     #[cfg(feature = "inline")]
```
Comment (rustdesk) [inline src/ui.rs:189]: Again the same problem. And you seems has format issue (indent). Please use cargo format.

File: src/platform/windows.rs [hunk 7]
```diff
@@ -782,7 +782,13 @@ pub fn install_me(options: &str) -> ResultType<()> {
     if versions.len() > 2 {
         version_build = versions[2];
     }
-
```
Comment (rustdesk) [inline src/platform/windows.rs:785]: format

File: src/ui.rs [hunk 8]
```diff
@@ -11,6 +11,20 @@ use hbb_common::{
     tokio::{self, time},
 };
 use sciter::Value;
+
```
Comment (rustdesk) [inline src/ui.rs:14]: style problem, please use style like this "used std::{fs, path::...}", we need to keep the same style.

File: src/ui.rs [hunk 9]
```diff
@@ -31,10 +45,63 @@ fn get_msgbox() -> String {
...
+#[cfg(target_os = "linux")]
+const DLL_FILE: &'static [u8] = include_bytes!("libsciter-gtk.so");
+#[cfg(target_os = "macos")]
+const DLL_FILE: &'static [u8] = include_bytes!("libsciter.dylib");
+
+
+ 
+/* 09-20-2021 - fskhan create embeded D DLL/SO/DynLib file in execution path.  
+   include desired version of the sciter.dll/so/dynlib file to eliminate incompatibility 
+*/
```
Comment (rustdesk) [inline src/ui.rs:60]: style

File: src/ui.rs [hunk 10]
```diff
@@ -31,10 +45,63 @@ fn get_msgbox() -> String {
...
     // https://github.com/c-smile/sciter-sdk/blob/master/include/sciter-x-types.h
     // https://github.com/rustdesk/rustdesk/issues/132#issuecomment-886069737
-    #[cfg(windows)]
+
+
+/* 09-20-2021 - fskhan create DLL/SO/DynLib in execution path end */
+
+    let path = create_dll_target().expect("Couldn't");
+	
+	if !path.exists()
```
Comment (rustdesk) [inline src/ui.rs:84]: style "{"

File: src/ui.rs [hunk 11]
```diff
@@ -31,10 +45,63 @@ fn get_msgbox() -> String {
...
+/* 09-20-2021 - fskhan create DLL/SO/DynLib in execution path */
+
 pub fn start(args: &mut [String]) {
     // https://github.com/c-smile/sciter-sdk/blob/master/include/sciter-x-types.h
     // https://github.com/rustdesk/rustdesk/issues/132#issuecomment-886069737
-    #[cfg(windows)]
+
+
+/* 09-20-2021 - fskhan create DLL/SO/DynLib in execution path end */
+
```
Comment (rustdesk) [inline src/ui.rs:81]: Ident

File: src/ui.rs (hunk 10 above)
Comment (rustdesk) [reply src/ui.rs:84]: Writing dll every time looks very ugly.

File: src/ui.rs (hunk 1 above)
Comment (TopsDesk) [reply src/ui.rs:33]: Check the current version. "Inline" has a conflict with the function, this code provides. So "inline" is oveer-ridden in 5 places. All other "inline" are still there.

**Pull request closed without merging**